client/my-sites/plans/plan-overview/plan-status/index.jsx

```diff
@@ -55,12 +81,14 @@ const PlanStatus = React.createClass( {
 								} )
 							}
 						</h1>
+						{ this.renderNotice() }
 					</div>
 
 					<Button
 						primary={ getDaysUntilUserFacingExpiry( this.props.plan ) < 6 }
 						className="plan-status__button"
-						onClick={ this.purchasePlan }>
+						onClick={ this.purchasePlan }
+					>
```

Comment:
Yes, this is a new rule in our [eslint config](https://github.com/Automattic/wp-calypso/blob/master/.eslintrc#L61).